### PR TITLE
bug: no pages in bibtex for book chapters

### PR DESCRIPTION
**Pull request by hanadikaleel**
#2144

### CODE REVIEW

File: backend/inspirehep/records/marshmallow/literature/bibtex.py [hunk 1]
```diff
@@ -300,10 +300,32 @@ def get_volume(self, data):
...
+        ]
+
+        if not only_publications:
+            return {}
+
+        return sorted(
+            only_publications, key=lambda item: item.get("year", float("inf"))
+        )[0]
+
     def get_pages(self, data):
```
Comment (michamos) [inline backend/inspirehep/records/marshmallow/literature/bibtex.py:320]: where is the test for this bugfix? did you try it out manually? how do you know it works?

File: backend/inspirehep/records/marshmallow/literature/bibtex.py [hunk 2]
```diff
@@ -300,10 +300,32 @@ def get_volume(self, data):
...
-        return LiteratureReader.get_page_artid_for_publication_info(
-            BibTexCommonSchema.get_best_publication_info(data), "--"
-        )
+        if "journal_title" in "publication_info":
```
Comment (drjova) [inline backend/inspirehep/records/marshmallow/literature/bibtex.py:321]: This will always be `False`. It is a string to string comparison.

Comment (hanadikaleel) [reply backend/inspirehep/records/marshmallow/literature/bibtex.py]: updated

File: backend/tests/unit/records/marshmallow/literature/test_bibtex_schema.py [hunk 3]
```diff
@@ -407,6 +407,28 @@ def test_get_page():
     assert expected_pages == result_pages
 
 
+def test_get_page_without_journal_title_volume():
```
Comment (drjova) [inline backend/tests/unit/records/marshmallow/literature/test_bibtex_schema.py:410]: what about a test with `page_start` and `page_end`?

**Comment by drjova** [comment]
@hanadikaleel could you please rebase?

**Review by drjova — APPROVED** [review]

**Pull request merged**